app/display/representation-javafx/src/main/java/org/csstudio/display/builder/representation/javafx/widgets/SpinnerRepresentation.java

```diff
@@ -352,6 +354,8 @@ protected void registerListeners()
         model_widget.propShowUnits().addUntypedPropertyListener(contentListener);
         model_widget.runtimePropValue().addUntypedPropertyListener(contentListener);
 
+        model_widget.propPVName().addPropertyListener(pvNameListener);

```

Comment:
call to addPropertyListener() should have corresponding removePropertyListener in `unregisterListeners` below.